ios/FluentUI/Button/Button.swift

```diff
@@ -221,6 +237,12 @@ open class Button: UIButton {
 
         titleLabel?.font = style.titleFont
         titleLabel?.adjustsFontForContentSizeCategory = true
+
+        if #available(iOS 15, *) {
+            var configuration = UIButton.Configuration.plain()
```

Comment:
why is this only time we set the self.configuration? any client updtes to edgeinset should trigger configuration update directly? it seems like only indirect call vias adjustCustomContentEdgeInsetsForImage() is handled

Reply:
before line 324 contentEdgeInset in update() alwasy did the right thing?